airflow/www/static/js/connection_form.js

```diff
@@ -62,6 +62,12 @@ $(document).ready(function () {
         'login': 'Username',
       }
     },
+    sentry: {
+      hidden_fields:['host', 'schema', 'login', 'password', 'port', 'extra'],
+      relabeling: {
+        'conn_id': 'dsn'
```

Comment:
That doesn't seem right - conn_id is a name to lookup the connection by (i.e. `sentry_default` or similar).

Sentry has an API key right? If so that should be stored in the password field, or the entire dsn should be in the `host` field.

Reply:
You are right, the previous file was a mistake, and I meant to place the dsn in the host field.